plugins/auth-backend/api-report.md

```diff
@@ -347,12 +345,6 @@ export type OAuthStartResponse = {
 // @public @deprecated (undocumented)
 export type OAuthState = OAuthState_2;
 
-// @public
-export type OidcAuthResult = {
```

Comment:
Let's switch this one over to just be deprecated instead, existing users might use it for their callbacks

Reply:
Sure thing is just adding the @deprecated tag enough or did you want some specific text in there as well?

Reply:
Just `@deprecated` is enough 👍